dockerfiles/Dockerfile.dpdk

```diff
@@ -0,0 +1,60 @@
+ARG UBUNTU_VERSION=20.04
+
+FROM ubuntu:$UBUNTU_VERSION
+
+ARG USERNAME=admin
+ARG PASSWORD=password
+ARG DPDK_VERSION=20.11.9
+ARG DPDK_FILE=dpdk-$DPDK_VERSION.tar.xz
+ARG DPDK_HOME=/home/$USERNAME/dpdk
+ARG BUILD_CPU_CORES=4
+
+# Set environment variables (optional)
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install necessary packages
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    libnuma-dev \
+    libpcap-dev \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+    python3-wheel \
+    ninja-build \
+    wget \
+    sudo \
+    git \
+    vim \
+    pkg-config \
+    && rm -rf /var/lib/apt/lists/* \
+    && pip3 install meson
+
+# Add a user with sudo privileges
+RUN useradd -m -s /bin/bash $USERNAME && \
+    echo "$USERNAME:$PASSWORD" | chpasswd && \

```

Comment:
If you want to unify the artifacts permission between docker and host, please create the docker user with the same uid and guid of host.
Here are an example:
https://github.com/sonic-net/sonic-buildimage/blob/a0e2082efd206876930dcd2b181a09963e892061/sonic-slave-jessie/Dockerfile.user.j2#L9-L11
https://github.com/sonic-net/sonic-buildimage/blob/a0e2082efd206876930dcd2b181a09963e892061/slave.mk#L9-L11
https://github.com/sonic-net/sonic-buildimage/blob/a0e2082efd206876930dcd2b181a09963e892061/slave.mk#L1009-L1019

Reply:
> If you want to unify the artifacts permission between docker and host, please create the docker user with the same uid and guid of host. Here are an example: https://github.com/sonic-net/sonic-buildimage/blob/a0e2082efd206876930dcd2b181a09963e892061/sonic-slave-jessie/Dockerfile.user.j2#L9-L11 https://github.com/sonic-net/sonic-buildimage/blob/a0e2082efd206876930dcd2b181a09963e892061/slave.mk#L9-L11 https://github.com/sonic-net/sonic-buildimage/blob/a0e2082efd206876930dcd2b181a09963e892061/slave.mk#L1009-L1019

Thanks a lot for the review, Ze Lao! I have updated the code to address your comments.